src/views/blocks/rsvp/content.php

```diff
@@ -23,6 +25,6 @@
 		<?php $this->template( 'blocks/rsvp/status', array( 'ticket' => $ticket, 'going' => $going ) ); ?>
 	</div>
 
-	<?php $this->template( 'blocks/rsvp/form', array( 'ticket' => $ticket, 'going' => $going ) ); ?>
+	<?php $this->template( 'blocks/rsvp/form', array( 'ticket' => $ticket, 'going' => $going, 'post_id' => $post_id ) ); ?>
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: f20752877dfc6adceec29972ff667653</sub>